_posts/en/2018-12-28-optech-review-2018-plan-2019.md

```diff
@@ -0,0 +1,79 @@
+  from member companies. Almost all attendees thought the workshops were ‘good’
+  or ‘excellent’, with an average score of 3.7/5 for the first and 4.2/5 for the
+  second.
+- Produced 1 pilot and 26 regular
+  [newsletters][]. If all the newsletters to date were printed out on typical
+  book pages, they'd be almost 100 pages long (33,000 words at 350 words per
+  page). Over 1,800 Bitcoin engineers and enthusiasts have subscribed to the
+  newsletter.
+- Built a [dashboard][] to track the state of the Bitcoin network and the
+  adoption rate of scaling technologies. Marcin wrote about the [building the
+  dashboard in an Optech blog post][dashboard blog post].
+- Started writing the first chapter of our [Scaling Book][scaling book].
+
+We think that’s a good start, and we’re all proud of what Optech has done so
+far, but we want to do even more next year. Optech's mission has not changed
+for 2019. We remain focused on helping companies adopt Bitcoin scaling
+technologies. Our high-level goals for 2019 are:
+
+1. Hold two workshops, maintaining the high standard of the previous two.
+   Following our recent workshop in Europe, we’re provisionally planning a US
+  East Coast workshop in the first half of 2019 and US West Coast in the second
+  half of 2019.
+1. Make significant progress on the Scaling Book. We want to complete the [fee
+   bumping chapter][scaling book feebumping] and several more chapters, and
+  have multiple contributors to future chapters.
+1. Publish six more field reports from engineers at member companies that
+   describe their experience implementing scaling techniques. We all really
+   enjoyed AJ Towns’s field report on UTXO consolidation in [Newsletter
+   #6][newsletter 6], and would love to see more reports.
+1. Add at least six more members in 2019. There are still a few large exchanges
+   that haven’t become members yet and we’d love to reach them.
+1. Publish a ‘compatibility matrix’ showing different wallets’ and exchanges’
+   support for scaling technologies. Mike has already been working on
+   documenting wallet and exchange compatibility with RBF and we plan to publish
+   that work on our website in early 2019.
+
+There are a few other areas we think we can help:
+
+- We want our newsletter to reach more people who don’t necessarily speak
+  English, particularly in China, Japan and Korea. If you’re a subscriber and
+  would like to help translating the newsletter, let us know!
+- As we move towards formalization and possible implementation of a
+  [Schnorr/Taproot softfork][softfork], we want to help our member companies
+  and other Bitcoin organizations prepare for the change. We think we can help
+  with soliciting industry input to the specification and building docs and tools
+  to help early adopters.
+
+Thank you to all of our sponsors, members and subscribers for your support this
+year! It really does mean a lot to us. As always, please contact us if you’d
```

Comment:
Suggestion: change contact us to an email link.  If you make the link `[contact us][email optech]` and add `{% included references.md %}` in the file, this will use our globally-defined address.

Reply:
done!